app/views/notifications/_sidebar.html.erb

```diff
@@ -29,8 +29,8 @@
 
   <%= menu_separator %>
 
-  <% @unread_notifications.each do |unread_notification, count| %>
-    <%= filter_link :unread, unread_notification, count do %>
+  <% [false, true].each do |unread_notification| %>
```

Comment:
It's a bit counter-intuitive to be `unread_notification` but be for read and unread.
I'm also wondering if the loop is more complex than it needs to be. Is the code duplication here better than having the loop and trying to figure out what `false` and `true` is?

Reply:
Agreed, thanks for the second pair of 👀

Reply:
Force pushed the simpler version 👍